CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 ## 2.1.0 [in progress]
+### Bug fixes
+1. [#190](https://github.com/influxdata/influxdb-client-go/issues/190) Fixed QueryTableResult.Next() failed to parse boolean datatype.
```

Comment:
Please, add a link to PR, not to issue..

Reply:
ok now.